data_architect_misc/data_transformer/configs/LATAM_Mexico/step1_transform_raw_investment_data.json

```diff
@@ -0,0 +1,121 @@
+[
+  {
+    "__comment__": "We decided to split Mexico data processing into 2 steps because although the raw data files have similar structures, they have different number of rows to skip (skiprows) in the beginning.",
+    "__comment__": "It does not seem like it is worth it to split into two config files just because of this ONE difference. But we did it because if we combine both steps in one config file, we will have to use a function to load another file (because transform module only allows us to load ONE file as a base file in the beginning.",
+    "__comment__": "We want to prevent people forgetting to update the name of the new raw file to load every month. So we split these steps into two.",
+
+    "__comment__": "If we pass None for 'sheet_name' parameter in reading Excel files, pandas will read ALL the sheets.",
+    "__comment__": "The equivalent of None in JSON file is null. That's why we are providing null below to read all the sheets in the file.",
+    "header": 55,
+    "skiprows": 57,
+    "input_delimiter": ";",
+    "input_encoding" : "Windows-1252",
+    "skipfooter": 0,
+
+    "write_output": true,
+    "output_folder_path": "./output/LATAM_Mexico/",
+    "output_file_name_prefix": "Transformed_Mexico_INV_20200701_20200731_",
+
+    "custom_transform_functions_file": "./transform_functions/latam_mexico_transform_functions.py",
+
+    "functions_to_apply": [
+      {
+        "function_name": "drop_columns_by_name_if_they_exist_in_dataframe",
+        "function_args": [[" "]]
+      },
+      {
+        "__function_comment__": "Excel file has empty columns. We need to drop them first",
+        "function_name": "drop_unnamed_columns"
+      },
+      {
+        "__function_comment__": "By the time we run this function, there should be only 13 columns total remaining in the raw data frame.",
```

Comment:
there should be only 24 columns. Either remove the comment or update the comments .

Please update the comments here to make sure they are all reflective of the Mexico processing. Thank you.